janus_server/src/aggregator.rs

```diff
@@ -439,11 +442,163 @@ where
 
     async fn handle_aggregate_continue(
         &self,
-        _task_id: TaskId,
-        _job_id: AggregationJobId,
-        _transitions: Vec<Transition>,
+        task_id: TaskId,
+        job_id: AggregationJobId,
+        transitions: Vec<Transition>,
     ) -> Result<AggregateResp, Error> {
-        todo!() // TODO(brandon): implement
+        let vdaf = Arc::new(self.vdaf.clone());
+        let verify_param = Arc::new(self.verify_param.clone());
+        let transitions = Arc::new(transitions);
```

Comment:
Sadly, I don't think so; I tried a number of approaches and Rust complained about lifetimes in every case.

This makes some sense to me: the closure passed to `run_tx` is not a `FnOnce` but a `Fn` (and in the face of transaction retries, we actually will call the closure multiple times). If we moved `transitions` into the closure, we couldn't call the closure more than once, since the first call would consume `transitions`. So it seems that we do need _some_ way to keep `transitions` around for multiple calls, and cloning an `Arc` is a lot cheaper than cloning the whole structure. Keep in mind that `Arc::new` is moving just the constant-size "header" of the transitions vector, not the contents of the vector.

I'm more concerned about the other `Arcs` TBH -- we end up cloning the VDAF & the verify parameter, with no guarantee that either clone is "cheap" since the size of these structures is up to the VDAF in use. I think we may want to refactor the `Aggregator` struct to hold onto an `Arc<A>` and an `Arc<A::VerifyParam>` to avoid these clones, but I opted not to do this performance enhancement in this PR.